src/main/java/com/google/opensesame/servlets/UserServlet.java

```diff
@@ -33,38 +37,33 @@ public void init() {
   }
 
   @Override
-  // Get a specific user. Return null if not found.
+  // Get a specific user. Return the currently signed-in user if no userID is
+  // supplied.
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String userGithub = request.getParameter("githubID"); // TODO:Check for null values
-    Key userKey = KeyFactory.createKey(PersonBuilder.ENTITY_NAME, userGithub);
 
-    Entity userEntity; // TODO: Abstract doGet() into it's own function
-    try {
-      userEntity = datastore.get(userKey);
-    } catch (EntityNotFoundException e) {
-      ErrorResponse.sendJsonError(
-          response,
-          "User not fount in the Datastore",
-          HttpServletResponse.SC_BAD_REQUEST,
-          "The user requested could not be found on the server."
-              + "Please ensure that the user has an account with us.");
-      return;
+    String userID = request.getParameter("userID");
+    if (userID == null) {
+      try {
+        userID = AuthServlet.getAuthorizedUser().getUserId();
+      } catch (NullPointerException e) {
+        ErrorResponse.sendJsonError(
+            response,
+            "UserID not Supplied and user not logged in",
+            HttpServletResponse.SC_FORBIDDEN,
+            "You are not logged in");
+        return;
+      }
```

Comment:
Could we use an if statement here instead? It keeps things much more readable imo, and it avoids an unnecessary null pointer exception:

```suggestion
      User currentUser = AuthServlet.getAuthorizedUser();
      if (currentUser == null) {
        ErrorResponse.sendJsonError(
            response,
            "UserID not Supplied and user not logged in",
            HttpServletResponse.SC_FORBIDDEN,
            "You are not logged in");
        return;
      } else {
        userID = currentUser.getUserId();
      }
```